production/helm/loki/templates/_helpers.tpl

```diff
@@ -454,6 +454,82 @@ ruler:
 {{- end }}
 {{- end }}
 
+{{/* Enterprise Logs Admin API storage config */}}
+{{- define "enterprise-logs.adminAPIStorageConfig" }}
+storage:
+  {{- if .Values.minio.enabled }}
+  backend: "s3"
+  s3:
+    bucketnames: admin
+  {{- else if eq .Values.loki.storage.type "s3" -}}
+  {{- with .Values.loki.storage.s3 }}
+  backend: "s3"
+  s3:
+    bucketnames: {{ $.Values.loki.storage.bucketNames.admin }}
```

Comment:
I think the admin storage config has different value names here? e.g. it uses `bucket_name` not `bucketnames`

Reply:
you are completely right;) let me fix it asap

Reply:
done